src/test/java/org/zalando/nakadi/validation/JSONSchemaValidationTest.java

```diff
@@ -18,105 +17,13 @@
 public class JSONSchemaValidationTest {
 
     static {
-        ValidationStrategy.register(EventBodyMustRespectSchema.NAME, new EventBodyMustRespectSchema());
+        ValidationStrategy.register(EventBodyMustRespectSchema.NAME, new EventBodyMustRespectSchema(
+                new JsonSchemaLoader()));
         ValidationStrategy.register(EventMetadataValidationStrategy.NAME, new EventMetadataValidationStrategy());
         ValidationStrategy.register(FieldNameMustBeSet.NAME, new FieldNameMustBeSet());
     }
 
     @Test
-    public void schemaValidationShouldRespectEventTypeDefinition() {
```

Comment:
Schema validation is a task of the everit library. We are not going to test it again in our project. It already has hundreds of tests in the library.